### PR TITLE
Update: improve_data_quality.ipynb Notebook 

### PR DESCRIPTION
**Pull request by kalyanipande-tudip**
- [x] Added the informative comments with all description in solution notebook.
- [x] Updated the content formatting of the instructions in solution notebook.
- [x] Removed the unnecessary instructions from the solution notebook.
- [x] Related Buganizer: http://b/170922782

### CODE REVIEW

File: courses/machine_learning/deepdive2/launching_into_ml/solutions/improve_data_quality.ipynb [hunk 1]
```diff
@@ -34,36 +34,10 @@
    "metadata": {},
```
Comment (gstripling) [inline courses/machine_learning/deepdive2/launching_into_ml/solutions/improve_data_quality.ipynb:34]: <p>I would suggest putting the #First, we'll import Pandas, a data processing and CSV file I/O above the line 3 (e.g. import pandas as pd).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1267/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (gstripling) [inline courses/machine_learning/deepdive2/launching_into_ml/solutions/improve_data_quality.ipynb:34]: <p>Line 2, add the word "the" after Using.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1267/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (gstripling) [inline courses/machine_learning/deepdive2/launching_into_ml/solutions/improve_data_quality.ipynb:34]: <p>Line 1, add the word "the" after using.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1267/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (gstripling) [inline courses/machine_learning/deepdive2/launching_into_ml/solutions/improve_data_quality.ipynb:34]: <p>Leave the original sentence here, e.g. First, let's summarize.....</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1267/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (gstripling) [inline courses/machine_learning/deepdive2/launching_into_ml/solutions/improve_data_quality.ipynb:34]: <p>Replace line 1, with this:  In Python&nbsp;shape()&nbsp;is used in pandas to give the number of rows/columns.  The number of rows is given by .shape[0].  The number of columns is given by .shape[1].  Thus,  <code>shape()</code>&nbsp;consists of an array having two arguments -- rows and columns</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1267/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Comment (gstripling) [inline courses/machine_learning/deepdive2/launching_into_ml/solutions/improve_data_quality.ipynb:34]: <p>Leave the original line, "Now, let's confirm the date parsing".</p><p> </p><p>Replace line 1 with, "Here, we are creating a new dataframe called "grouped_data" and grouping by on the column "Make".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1267/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Comment (gstripling) [inline courses/machine_learning/deepdive2/launching_into_ml/solutions/improve_data_quality.ipynb:34]: <p>Line 1, Replace with this:  Here we will visualize our data using the figure() function in the pyplot module of matplotlib's library -- which is used to create a new figure.</p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1267/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Comment (gstripling) [inline courses/machine_learning/deepdive2/launching_into_ml/solutions/improve_data_quality.ipynb:34]: <p>Line 1, replace with with:  &nbsp;Let's convert the Yes to 1 and No to 0. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1267/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Comment (kalyanipande-tudip) [reply courses/machine_learning/deepdive2/launching_into_ml/solutions/improve_data_quality.ipynb:34]: Updated the same.

Comment (kalyanipande-tudip) [reply courses/machine_learning/deepdive2/launching_into_ml/solutions/improve_data_quality.ipynb:34]: Updated the same.

Comment (kalyanipande-tudip) [reply courses/machine_learning/deepdive2/launching_into_ml/solutions/improve_data_quality.ipynb:34]: Updated the same.

Comment (kalyanipande-tudip) [reply courses/machine_learning/deepdive2/launching_into_ml/solutions/improve_data_quality.ipynb:34]: Updated the same.

Comment (kalyanipande-tudip) [reply courses/machine_learning/deepdive2/launching_into_ml/solutions/improve_data_quality.ipynb:34]: Updated the same.

Comment (kalyanipande-tudip) [reply courses/machine_learning/deepdive2/launching_into_ml/solutions/improve_data_quality.ipynb:34]: Updated the same.

Comment (kalyanipande-tudip) [reply courses/machine_learning/deepdive2/launching_into_ml/solutions/improve_data_quality.ipynb:34]: Updated the same.

Comment (kalyanipande-tudip) [reply courses/machine_learning/deepdive2/launching_into_ml/solutions/improve_data_quality.ipynb:34]: Updated the same.

**Comment by kalyanipande-tudip** [comment]
Hi @gstripling,

I have updated the changes suggested above and PR is ready to merge now.

Please review the same and feel free to let me know if any further changes are required here.

Thanks!

**Pull request merged**